### PR TITLE
Update configure-real-time-protection-microsoft-defender-antivirus.md

### PR DESCRIPTION
**Pull request by msbemba**
Added max value for Define the maximum size of downloaded files and attachments to be scanned

Per documents 
Default Value	20480
Min Value	0
Max Value	10000000

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5522

File: microsoft-365/security/defender-endpoint/configure-real-time-protection-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ To enable and configure always-on protection:
        | Monitor file and program activity on your computer | The Microsoft Defender Antivirus engine makes note of any file changes (file writes, such as moves, copies, or modifications) and general program activity (programs that are opened or running and that cause other programs to run). | Enabled |
        | Turn on raw volume write notifications | Information about raw volume writes will be analyzed by behavior monitoring. | Enabled |
        | Turn on process scanning whenever real-time protection is enabled | You can independently enable the Microsoft Defender Antivirus engine to scan running processes for suspicious modifications or behaviors. This is useful if you have temporarily disabled real-time protection and want to automatically scan processes that started while it was disabled. | Enabled |
-       | Define the maximum size of downloaded files and attachments to be scanned | You can define the size in kilobytes. | Enabled |
+       | Define the maximum size of downloaded files and attachments to be scanned | You can define the size in kilobytes. Max Value is 10000000.  | Enabled |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-real-time-protection-microsoft-defender-antivirus.md]:
```suggestion
       | Define the maximum size of downloaded files and attachments to be scanned | You can define the size in kilobytes. The maximum value is 10,000,000.  | Enabled |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Comment by denisebmsft** [comment]
@msbemba, can you please tell me where you got the details? I am asking because one of the PMs is telling me that limit does not exist starting with MOCAMP 1908, and I want to make sure it's correct.

**Comment by denisebmsft** [comment]
Checked with my PMs and we are going to decline to implement this change at this time.

**Comment by msbemba** [comment]
@denisebmsft The max value I got online for the registry key for Real-time protection is as mentioned below. So that's where I got this information. This could very well be outdated, So can we say there is no max size limit as such?

image

**Comment by HotCakeX** [comment]
> @denisebmsft The max value I got online for the registry key for Real-time protection is as mentioned below. So that's where I got this information. This could very well be outdated, So can we say there is no max size limit as such?
> 
> image


Hi,
The Group Policy setting in Windows 11 22H2 also has the same 10,000,000 limit which is 10GB, doesn't go beyond that.

image

**Pull request closed without merging**